scripts/poetry.lock

```diff
@@ -2,21 +2,22 @@
 
 [[package]]
 name = "attrs"
-version = "23.1.0"
+version = "23.2.0"
```

Comment:
Following [the scripts readme](https://github.com/opengeospatial/geoparquet/blob/main/scripts/README.md#usage), I ran `poetry update`.  This resulted in the lockfile changes below.  If we don't want this to happen, we should update that readme.